prow/cmd/peribolos/OWNERS

```diff
@@ -1,16 +1,15 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-- cblecker
 - cjwagner
 - MadhavJivrajani
-- nikhita
 - Priyankasaggu11929
 approvers:
-- cblecker
```

Comment:
I'm fine remaining a peribolos approver :) (this is unrelated to the TL role)

Reply:
🎉 
Added back!

Reply:
I'd like to remain here as well! Will move myself to emeritus in a few months if I'm unable to spend time here

Reply:
Yay!
Added back ❤️